cmake/sip_configure.py

```diff
@@ -2,7 +2,49 @@
 import re
 import subprocess
 import sys
-from PyQt4 import pyqtconfig
+import sipconfig
+try:
+    from PyQt4.pyqtconfig import Configuration
+except ImportError:
+    # PyQt>=4.11 has no pyqtconfig
+    import PyQt4
+    from PyQt4 import QtCore
+    class Configuration(sipconfig.Configuration):
+        """The class that represents PyQt configuration values.
+        """
+        def __init__(self):
+            qtconfig = subprocess.check_output(["qmake-qt4", "-query"], universal_newlines=True)
+            qtconfig = dict(x.split(":", 1) for x in qtconfig.splitlines())
+            
+            pyqtconfig = {
+                "pyqt_config_args":   "--confirm-license -v /usr/share/sip --qsci-api -q /usr/bin/qmake-qt4",
+                "pyqt_version":       QtCore.PYQT_VERSION,
+                "pyqt_version_str":   QtCore.PYQT_VERSION_STR,
+                "pyqt_bin_dir":       PyQt4.__path__[0],
+                "pyqt_mod_dir":       PyQt4.__path__[0],
+                "pyqt_sip_dir":       "/usr/share/sip",
```

Comment:
This fails on Arch. `/usr/share/sip` contains several directories:
* `Py2-PyQt4`
* `Py2-PyQt5`
* `PyQt4`
* `PyQt5`
* `Qwt5`

I had to change `"pyqt_sip_dir"` to `"/usr/share/sip/Py2-PyQt4"` to fix the compilation of rviz.